### PR TITLE
[housekeeping] Cleanup code format

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change

Cleanup code format

### Issues Fixed

None

### CODE REVIEW

File: src/Compatibility/Core/src/Android/FastRenderers/ButtonRenderer.cs [hunk 1]
```diff
@@ -252,7 +252,7 @@ protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEv
 			{
 				UpdateTextColor();
 			}
-			else if (   e.PropertyName == FontElement.FontAttributesProperty.PropertyName
+			else if (e.PropertyName == FontElement.FontAttributesProperty.PropertyName
```
Comment (StephaneDelcroix) [inline src/Compatibility/Core/src/Android/FastRenderers/ButtonRenderer.cs:255]: this is on purpose so the conditions align

File: src/Compatibility/Core/src/iOS/Renderers/ButtonRenderer.cs [hunk 2]
```diff
@@ -137,7 +137,7 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 			if (e.PropertyName == Button.TextColorProperty.PropertyName)
 				UpdateTextColor();
-			else if (   e.PropertyName == FontElement.FontAttributesProperty.PropertyName
+			else if (e.PropertyName == FontElement.FontAttributesProperty.PropertyName
```
Comment (StephaneDelcroix) [inline src/Compatibility/Core/src/iOS/Renderers/ButtonRenderer.cs:140]: same

File: src/Controls/src/Core/BindableObject.cs [hunk 3]
```diff
@@ -446,7 +446,7 @@ internal void SetValueCore(BindableProperty property, object value, SetValueFlag
 			// else omitted on purpose
 
 			//if we're updating a dynamic resource from style, set the default backup value
-			if (   (context.Attributes & (BindableContextAttributes.IsDynamicResource | BindableContextAttributes.IsSetFromStyle)) != 0
+			if ((context.Attributes & (BindableContextAttributes.IsDynamicResource | BindableContextAttributes.IsSetFromStyle)) != 0
```
Comment (StephaneDelcroix) [inline src/Controls/src/Core/BindableObject.cs:449]: this is on purpose so the conditions aligns

File: src/Controls/src/Xaml/ApplyPropertiesVisitor.cs [hunk 4]
```diff
@@ -129,7 +129,7 @@ public void Visit(ElementNode node, INode parentNode)
 				Exception xpe = null;
 
 				string xKey = null;
-				if (   xpe == null
+				if (xpe == null
```
Comment (StephaneDelcroix) [inline src/Controls/src/Xaml/ApplyPropertiesVisitor.cs:132]: same

File: src/Controls/src/Xaml/ApplyPropertiesVisitor.cs [hunk 5]
```diff
@@ -139,12 +139,12 @@ public void Visit(ElementNode node, INode parentNode)
 				}
 
 				//ResourceDictionary
-				if (   xpe == null
+				if (xpe == null
```
Comment (StephaneDelcroix) [inline src/Controls/src/Xaml/ApplyPropertiesVisitor.cs:142]: same

File: src/Controls/src/Xaml/ApplyPropertiesVisitor.cs [hunk 6]
```diff
@@ -139,12 +139,12 @@ public void Visit(ElementNode node, INode parentNode)
...
 					&& TryAddToResourceDictionary(source as ResourceDictionary, value, xKey, node, out xpe))
 					return;
 
 				//ContentProperty
-				if (   xpe == null
+				if (xpe == null
```
Comment (StephaneDelcroix) [inline src/Controls/src/Xaml/ApplyPropertiesVisitor.cs:147]: same

File: src/Controls/src/Xaml/ApplyPropertiesVisitor.cs [hunk 7]
```diff
@@ -158,7 +158,7 @@ public void Visit(ElementNode node, INode parentNode)
 				}
 
 				// Collection element, implicit content, or implicit collection element.
-				if (   xpe == null
+				if (xpe == null
```
Comment (StephaneDelcroix) [inline src/Controls/src/Xaml/ApplyPropertiesVisitor.cs:161]: same

**Review by StephaneDelcroix — CHANGES_REQUESTED** [review]

**Comment by mattleibow** [comment]
Are you going to fight the machines on this?

**Comment by Eilon** [comment]
> Are you going to fight the machines on this?

😁 My personal view is don't fight the machines. Even if we undo this now, the next time someone runs the tool no one will notice and it will get changed anyway.

**Pull request closed without merging**